lms/tasks/email_digests.py

```diff
@@ -144,8 +146,10 @@ def send_instructor_email_digest(
 
             if task_done_data:
                 created_after = max(
-                    datetime.fromisoformat(task_done_data["created_before"]),
-                    created_after,
+                    datetime.fromisoformat(task_done_data["created_before"]).replace(
+                        tzinfo=timezone.utc
+                    ),
+                    created_after.replace(tzinfo=timezone.utc),
                     # Don't count annotations from before we deployed https://github.com/hypothesis/lms/pull/5904.
                     # This line can safely be removed on Weds 20th Dec 2023 or later.
                     datetime(year=2023, month=12, day=13, hour=5, tzinfo=timezone.utc),
```

Comment:
We could have just removed this tzinfo here but I reckon that's more brittle.

Reply:
Yeah, I thought it was best to always deal with timezone-aware, UTC datetimes (but unfortunately missed one of them)